arangodb-net-standard/Serialization/ApiClientSerialization.cs

```diff
@@ -12,7 +12,7 @@ public abstract class ApiClientSerialization : IApiClientSerialization
         /// The default serialization options.
         /// </summary>
         public virtual ApiClientSerializationOptions DefaultOptions { get; } =
-            new ApiClientSerializationOptions(false, true, false);
+            new ApiClientSerializationOptions(false, true, false, true);
```

Comment:
As the number of parameters is growing, a reading improvement would be to use named parameters:

```c#
new ApiClientSerializationOptions(
    useCamelCasePropertyNames: false,
    ignoreNullValues: true,
    useStringEnumConversion: false,
    ignoreMissingMember: true);
```

Not a blocker.

Reply:
Thanks for the suggestion. I have added it.